l10n_it_central_journal_reportlab/wizard/print_giornale.xml

```diff
@@ -8,7 +9,7 @@
             <form string="General Journal">
                 <separator string="Dates movements" />
                 <group>
-                    <field name="daterange_id" />
+                    <field name="daterange_id" domain="[('company_id', '!=', False)]" />
```

Comment:
Puoi aggiungere un commento spiegando la ragione per cui è stato aggiunto questo domain? (Mi ricordo che se ne era parlato in call). Grazie.